wordpress-mobile/WordPress-Android
# Issue/revisions user details

This PR adds functionality for retrieving details of revision authors and injecting them into the revision list.

After we done retrieving revisions, we gather ID of revision authors, and use them to retirieve their details by utilizing `/batch/`  and `/sites/$site/users` endpoing with search parameter.

Utilizing `/batch/` endpoint is a bit hacky, but it works. 

To test:
1. Open Revision for a post with multiple authors.
2. Make sure their avatars are displayed properly.

PS: This PR does not incliudes the Display Names of Revision authors. @iamthomasbishop If you would like to add it to the design let me know :)

[![Image from Gyazo](https://i.gyazo.com/6f804cbcdc597894288a7d2f121ce7fb.png)](https://gyazo.com/6f804cbcdc597894288a7d2f121ce7fb)
